Fix tilemap gaps in affine graphic modes #733 (#735)
agbrs/agb: CHANGELOG.md

## [Unreleased]

### Fixed

> - Kludge fix for tilemap gaps in affine graphics modes.

Reviewer: ```suggestion
- There are no longer gaps between tiles in affine graphics modes.
```
